website/source/intro/getting-started/jobs.html.md

```diff
@@ -215,12 +228,12 @@ We can run `plan` again to see what will happen if we submit this change:
 ```
 $ nomad plan example.nomad
 +/- Job: "example"
-+/- Task Group: "cache" (3 create/destroy update)
++/- Task Group: "cache" (1 create/destroy update, 2 ignore)
```

Comment:
note to @dadgar -- need to update prose below this example to reflect the change in output.